scripts/post_changelog_to_public_documentation_rss_feed.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+# fail if any commands fails
+set -e
+
+# debug log
+set -x
+
+# privacy_view options:
+README_HIDE_PUBLICATION="anyone_with_link"
+README_PUBLIC_PUBLICATION="public"
```

Comment:
Proposal: make the variable names clearer.

```suggestion
README_RSS_ENTRY_VISIBILITY_PRIVATE="anyone_with_link"
README_RSS_ENTRY_VISIBILITY_PUBLIC ="public"
```

_Don't forget to change names at usage too_